tasks/main.yml

```diff
@@ -24,7 +30,7 @@
   copy:
     dest: /etc/acme-dns-pdns.yml
     content: "{{ config }}"
-    owner: root
+    owner: acme-dns-pdns
```

Comment:
Why should the service be able to modify its own config? `0640` already implies that it can read the file

Reply:
This is what we talked about. Somehow it doesn't suffice that the service runs with the right group. We could use 440 permissions though?